app/src/main/java/com/my/journeyplanner/views/itineraryresults/ItineraryResultsCardviewFragment.kt

```diff
@@ -0,0 +1,82 @@
+package com.my.journeyplanner.views.itineraryresults
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.my.core.domain.JourneyPlannerResultDomainModel.Itinerary.Journey
+import com.my.journeyplanner.R
+import com.my.journeyplanner.helpers.MarginItemDecoration
+import com.my.presenter.itineraryresults.ItineraryResultsCardviewFragmentContract
+import mu.KotlinLogging
+import java.io.Serializable
+
+const val DIVIDER_SPACE_HEIGHT = 32
+private val logger = KotlinLogging.logger {}
+
+class ItineraryResultsCardviewFragment : Fragment(), ItineraryResultsCardviewFragmentContract.View {
```

Comment:
it doesn't really matter that it's a Cardview?

Reply:
Renamed to ItineraryResultsFragment after clearing out old class of the same name